tests/functional-tests/src/test/groovy/specs/MinesOperatorSpec.groovy

```diff
@@ -1,80 +1,82 @@
 package specs
 
-import geb.spock.GebReportingSpec
-import listeners.BrowserStackReportingSpec
+import geb.spock.GebSpec
+import spock.lang.Stepwise
 import pages.MinesOperatorPage
 import org.openqa.selenium.Keys
-//GebReportingSpec
+import specs.traits.Utils
 
-
-class MinesOperatorSpec extends BrowserStackReportingSpec {
+class MinesOperatorSpec extends BaseSpec implements Utils {
 
     def "As a user, I want to fill in the Mines Operator Form."() {
         when: 'I enter the url for the Form'
           to MinesOperatorPage
-          InjectjQuery()
+          injectjQuery()
         then: 'I arrive at the correct page'
           at MinesOperatorPage
         when: 'I click on the -Go to Step 2- Button'
           NextStep()
         then: 'I see the second page'
-          assert lowerTitle
+          //assert lowerTitle
 
           //Registered Business Name
-          registeredBusinessName.value("STENS CONSULTING INC.")
+          setRegisteredBusinessName(randomBusinessName())
 
           //Primary Contact
-          firstName.value("First Name")
-          lastName.value("Last Name")
-          phoNe1.value("1234567890")
-          phoNe2.value("3335551234")
-          eMail.value("fake@email.com")
+          setFirstName(randomFirstName())
+          setLastName(randomLastName())
+          setPhone(randomPhoneNumber())
+          setAlternativePhone(randomPhoneNumber())
+          setEmail(randomEmail())
 
           //Business Address
-          businessAddressLine1.value("BA Line 1")
-          businessAddressLine2.value("BA line 2")
-          businessAddressCity.value("BA City")
-          businessAddressProvince << "AB"
-          businessAddressPostalCode.value("X9X9X9")
+          businessAddressLine1.value(randomStreetName())
+          businessAddressLine2.value(randomStreetName())
+          businessAddressCity.value(randomCityName())
+          businessAddressProvince << randomProvince()
+          businessAddressPostalCode.value(randomPostalCode())
 
           //COVID-19 Coordinator
-          covidFirstName.value("covid First Name")
-          covidLastName.value("covid Last Name")
-          covidPhone1.value("123456789")
-          covidPhone2.value("9087654321")
-          covidEmail.value("covid@email.com")
+          covidFirstName.value(randomFirstName())
+          covidLastName.value(randomLastName())
+          covidPhone1.value(randomPhoneNumber())
+          covidPhone2.value(randomPhoneNumber())
+          covidEmail.value(randomEmail())
 
           //Provide your accommodation details
           startDate.click()
-          //Thread.sleep(500)
-          startDate.jquery.removeAttr("readonly") //Nasty Hack to make the dates working
-          startDate << "2020-06-15"
+          basestartDate.jquery.removeAttr("readonly") //Nasty Hack to make the dates working
+          basestartDate << "2020-07-15"
+          basestartDate.jquery.attr("readonly","readonly") //Nasty Hack to make the dates working
```

Comment:
Does the infrastructure have the ability to run the more JS standard .addAttribute() and .removeAttribute() functions or does it need to leverage jQuery to work?

Reply:
Standard JS does not work in the test framework for this purpose. The jQuery function does not get all confused by Vue.
I'd like to have the JS possibility, but after I spent a full day on trying to make that work, I resigned to using jQuery.